low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -191,7 +191,7 @@ class SSPLDevDeploy:
 
```

Comment:
cortx-py-utils needs paramiko==**2.7.1**
Please bring this change as well along with this.

Reply:
done. 👍